code/modules/client/client_defines.dm

```diff
@@ -1,10 +1,10 @@
 /client
-		//////////////////////
-		//BLACK MAGIC THINGS//
-		//////////////////////
-	parent_type = /datum
+	parent_type = /datum		// black magic
+	preload_rsc = PRELOAD_RSC	// This is 0 so we can set it to an URL once the player logs in and have them download the resources from a different server.
+	view = WORLD_VIEW
+	var/view_radius = WORLD_VIEW_NUM
```

Comment:
how does this differ from view? Doesn't it do the same thing temp_view did?

Reply:
> how does this differ from view? Doesn't it do the same thing temp_view did?

view_radius <=> temp_view